terratorch/models/wxc_model_factory.py

```diff
@@ -43,4 +52,5 @@ def build_model(
         **kwargs,
     ) -> Model:
         module = get_finetune_model(kwargs['model_config'])
+        logger.info("Using wrapper.")
```

Comment:
is this necessary / helpful? it will always be printed, right?

Reply:
It was just useful during the debugging. I'll remove it.